site/docs/plugins/emoji.md

```diff
@@ -1 +1,68 @@
-Coming soon, please come back later
+# Emoji Plugin (`emoji`)
+
+With this plugin, you can easily insert emojis on your replies searching for them and inserting text instead of manually copying an emoji from web or Telegram and pasting at your code.
+
+## Installing and examples
+
+You can install this plugin on your bot like this:
+
+<CodeGroup>
+  <CodeGroupItem title="TypeScript" active>
+
+```ts
+import { EmojiFlavor, emojiParser } from "@grammyjs/emoji";
+
+type FlavoredContext =
```

Comment:
Maybe call it `MyContext` like all other plugins? Optionally, provide a relative link to the context flavouring if you want.